docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -131,38 +126,32 @@ done
 . Get the count of not-OK source databases:
 +
 ----
-oc rsh mariadb-copy-data mysql -h "${SOURCE_MARIADB_IP}" -uroot -p"${SOURCE_DB_ROOT_PASSWORD}" -e "SHOW databases;"
+$ oc rsh mariadb-copy-data mysql -h "${SOURCE_MARIADB_IP}" -uroot -p"${SOURCE_DB_ROOT_PASSWORD}" -e "SHOW databases;"
 ----
 
-. Run mysqlcheck on the original DB to look for things that are not OK:
+. Check that `mysqlcheck` had no errors:
 +
 ----
 . ~/.source_cloud_exported_variables
 test -z "$PULL_OPENSTACK_CONFIGURATION_MYSQLCHECK_NOK"  || [ "$PULL_OPENSTACK_CONFIGURATION_MYSQLCHECK_NOK" = " " ]
 ----
 
-. Test connection to control plane DBs (show databases):
+. Test the connection to the control plane databases:
 +
 ----
-oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
+$ oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
     mysql -rsh "$PODIFIED_MARIADB_IP" -uroot -p"$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;'
-oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
+$ oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
     mysql -rsh "$PODIFIED_CELL1_MARIADB_IP" -uroot -p"$PODIFIED_DB_ROOT_PASSWORD" -e 'SHOW databases;'
 ----
 +
 [NOTE]
-You need to transition {compute_service_first_ref} services imported later on into a
-superconductor architecture. For that, delete the old service records in
-cells DBs, starting from the cell1. New records will be registered with
-different hostnames provided by the {compute_service} service operator. All {compute_service}
-services, except the compute agent, have no internal state, and its service
-records can be safely deleted. You also need to rename the former `default` cell
-to `cell1`.
+You must transition {compute_service_first_ref} services imported later on into a superconductor architecture by deleting the old service records in cells databases, starting with `cell1`. New records are registered with different hostnames provided by the {compute_service} operator. All Compute services, except the Compute agent, have no internal state, and its service records can be safely deleted. You also need to rename the former `default` cell to `cell1`.
```

Comment:
You must transition {compute_service_first_ref} services imported later on... > You must transition {compute_service_first_ref} services that you import later...

or

You must transition {compute_service_first_ref} services that are imported later...